final_prep/annotations.rb

```diff
@@ -4,19 +4,19 @@
 
 # Build a Bear
 
-def build_a_bear(name, age, fur, clothes, special_power)
-  greeting = "Hey partner! My name is #{name} - will you be my friend?!"
-  demographics = [name, age]
-  power_saying = "Did you know that I can #{special_power}?"
+def build_a_bear(name, age, fur, clothes, special_power) #initializes a method that takes 5 inputs.
+  greeting = "Hey partner! My name is #{name} - will you be my friend?!" # Creates a variable named greeting, a string. Uses interpolation to put one of inputs into string.
+  demographics = [name, age] # creates an array variable
+  power_saying = "Did you know that I can #{special_power}?" #creates a string variable, with string interpolation
   built_bear = {
     'basic_info' => demographics,
     'clothes' => clothes,
     'exterior' => fur,
     'cost' => 49.99,
     'sayings' => [greeting, power_saying, "Goodnight my friend!"],
     'is_cuddly' => true,
-  }
-  return built_bear
+  } # creates a hash of a built bear.
+  return built_bear # 'saves' the information to the method
 end
 
 build_a_bear('Fluffy', 4, 'brown', ['pants', 'jorts', 'tanktop'], 'give you nightmares')
```

Comment:
Where is the annotation for this line? Don't forget the most important one! You could say something like "call the function and pass in 5 arguments"